osu.Game/Users/Drawables/ClickableAvatar.cs

```diff
@@ -67,5 +57,25 @@ protected override bool OnClick(ClickEvent e)
 
             return base.OnClick(e);
         }
+
+        private partial class UserGridPanelTooltip : VisibilityContainer, ITooltip<UserGridPanel>
+        {
+            private UserGridPanel? displayedUser;
+
+            protected override void PopIn()
+            {
+                Child = displayedUser;
```

Comment:
This is a potential null reference (shows up locally for me, but for some reason not on CI?).

I think you're going to want test coverage of the `ClickableAvatar` being used with a null user, and checking that the placeholder case behaves correctly.

Reply:
I wrote tests for the case if the APIUser is null, if that's what you meant. They just simply won't show up.

Did you meant unit tests under test coverage?